### PR TITLE
UAR-708: Remove (NOT LIVE) from page titles

### PR DESCRIPTION
**Pull request by mgibson-ch**
### JIRA link
https://companieshouse.atlassian.net/browse/UAR-708

### Change description
The content (NOT LIVE) has been removed from all page titles within the Update journey. Pages include (seen in Reserve1):
- Do any beneficial owners or managing officers have their personal information protected at Companies House?
- You'll need to file an update using the paper form (protected information stop screen)
- Date of the update statement
- Who can we contact about this update? 
- Who is completing this update?
- Complete this statement to confirm that verification checks have been completed
- Tell us about the UK-regulated agent that carried out verification checks 
- Check the overseas entity details
- Beneficial owners and managing officers involved in the overseas entity
- Beneficial Owner and Managing Officer add/review pages (individual BO, OLE BO, gov BO, individual MO, corporate MO)
- Check your answers before sending your application
- Confirmation

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by rafahop — APPROVED** [review]
Looks good to me

**Review by RobertBromfield — APPROVED** [review]
Usually wouldn't approve if SonarQube was failing

But as this is just getting rid of a substring many times it's obvious that code duplication was already there before this PR

File: src/controllers/update/confirmation.controller.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       verificationCheckDays: 14,
       paymentFee: UPDATE_PAYMENT_FEE,
       isUpdate: true,
-      titleExtension: "(NOT LIVE)",
+      titleExtension: "",
```
Comment (AndreKramer7654) [inline src/controllers/update/confirmation.controller.ts:31]: maybe we can remove titleExtension from the view?

Comment (mgibson-ch) [reply src/controllers/update/confirmation.controller.ts]: Done.

**Pull request merged**